src/table/context/createSelectorProvider.tsx

```diff
@@ -9,6 +9,7 @@ import React, {
   useRef,
 } from "react";
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Could not come up with a better type solution so had to stick with `any`

Reply:
yupp this is super general, so it is basically any